syfertext/pointers/token_pointer.py

```diff
@@ -26,8 +24,6 @@ def __init__(
             owner=owner,
             id=id,
             garbage_collect_data=garbage_collect_data,
-            tags=tags,
-            description=description,
         )
 
     @property
```

Comment:
As I said in the previous review, I think the return type of below `text_` property is a native `str` not a Syft String